Casks/gu-base.rb

```diff
@@ -0,0 +1,50 @@
+cask 'gu-base' do 
+  version '0.0.1'
+  sha256 :no_check
+
+  name 'Guardian Base'
+  homepage 'https://github.com/guardian/homebrew-devtools'
+  url 'https://github.com/guardian/homebrew-devtools'
+  stage_only true
+
+  # main applications
+  depends_on formula:  'openssl'
+  depends_on formula:  'mas'
+  depends_on formula:  'wget'
+  depends_on formula:  'ack'
+  depends_on formula:  'jq'
+  depends_on formula:  'bash-completion'
+  depends_on formula:  'coreutils'
+  depends_on formula:  'git'
+  depends_on formula:  'hub'
+  depends_on formula:  'gnupg'
+  depends_on formula:  'htop'
+  depends_on formula:  'mtr'
+  depends_on formula:  'tree'
+  depends_on formula:  'unrar'
+  depends_on formula:  'watch'
+
+  # dev langs
+  depends_on cask: 'caskroom/versions/java8'
+  depends_on formula:  'scala'
+  depends_on formula:  'sbt'
+  depends_on formula:  'node'
```

Comment:
`nvm` rather than `node`

Reply:
nvm will coexist happily with system node, and yarn works best (apparently) when installed with brew on OSX. 

I did forget to install `yarn` though.